### PR TITLE
Renamed TelemData folder and class to Pixhawk.

### PR DESCRIPTION
**Pull request by logankuzyk**
https://github.com/uvic-aero/onboard-computer/issues/7

Renamed TelemData and class to Pixhawk. The telemData.py file has not been renamed.

### CODE REVIEW

**Comment by dragonprevost** [comment]
Hey Logan, great work on the processes of creating a feature branch, making changes, pushing and getting used to the github workflow. 

A couple things that should be updated before this gets merged into master. 
* It seems that there is still an import that needs to be updated in `apps/Pixhawk/routes.py`.
```
 File "/Users/dragon/aero/onboard-computer/apps/Pixhawk/routes.py", line 1, in <module>
 from apps.TelemData.handlers import Status, Stop, Start
ModuleNotFoundError: No module named 'apps.TelemData.handlers'
```
*  I think you may have changed the execStart path in `obc.service` to match your machine. This may have happened automatically but just make sure to change it back to what it was. Also let me know if it happened automatically because it may be an issue on other peoples machines if that is the case

P.S. Stay hydrated and well fed!

File: obc.py [hunk 1]
```diff
@@ -23,7 +23,12 @@ def start(self):
 
         # start apps
         imageService.start()
+<<<<<<< HEAD
```
Comment (avesk) [inline obc.py]: Looks like there actually was some merge conflicts in this file that we missed. You're gonna wanna go in here and remove `<<<<<<< HEAD`, `=======` and `>>>>>>> 4fc1f82be6b1373c222d146900e064766842638e`

File: obc.py [hunk 2]
```diff
@@ -23,7 +23,12 @@ def start(self):
...
+        videoDisplay.start()
+        pixhawk.start()
+=======
         telemData.start()
```
Comment (avesk) [inline obc.py]: You're also gonna wanna take out this line as this is the "incoming changes", and we wanna keep your code which calls `pixhawk.start()`

File: obc.py [hunk 3]
```diff
@@ -23,7 +23,12 @@ def start(self):
 
         # start apps
         imageService.start()
+<<<<<<< HEAD
+        videoDisplay.start()
```
Comment (avesk) [inline obc.py]: This line should be removed as well since there is no longer a `videoDisplay` import

File: obc.service [hunk 4]
```diff
@@ -4,7 +4,7 @@ After=network.target
 
 [Service]
 Type=simple
-ExecStart=/home/pi/.virtualenvs/cv/bin/python dir/obc.py
```
Comment (avesk) [inline obc.service]: We're actually gonna want to keep this file the way it is. If you do a `git fetch` followed by a `git checkout master -- obc.service` that will revert your branche's `obc.service` to what's on master

Comment (avesk) [reply obc.service]: You'll have to run a `git commit -a` after that as well

File: obc.py [hunk 5]
```diff
@@ -33,7 +38,12 @@ def stop(self):
         # stop apps
         self.imageService.stop()
         self.timelapse.stop()
+<<<<<<< HEAD
```
Comment (avesk) [inline obc.py]: Similar thing here as above. Please remove `<<<<<<< HEAD`, `=======` and `>>>>>>> 4fc1f82be6b1373c222d146900e064766842638e`

File: obc.py [hunk 6]
```diff
@@ -33,7 +38,12 @@ def stop(self):
...
+        self.videoDisplay.stop()
```
Comment (avesk) [inline obc.py]: Remove this line for the reasons as on line 27

File: obc.py (hunk 3 above)
Comment (avesk) [reply obc.py]: @logankuzyk Heads up, I think you still have to remove `videoDisplay.start()`

File: obc.py (hunk 6 above)
Comment (avesk) [reply obc.py]: @logankuzyk Same thing here with `videoDisplay.start()`

File: obc.service (hunk 4 above)
Comment (avesk) [reply obc.service]: Looks like the Exec path is still changed here. This is a bit of a tricky fix if you do it the way I told you to. But that's the safest way to revert a change

You can also just replace `ExecStart=/home/pi/.virtualenvs/cv/bin/python /mnt/c/Users/logan/OneDrive/Documents/Aero/onboard-computer/obc.py` 
with
`ExecStart=/home/pi/.virtualenvs/cv/bin/python dir/obc.py`

File: obc.py [hunk 7]
```diff
@@ -23,7 +23,7 @@ def start(self):
 
         # start apps
         imageService.start()
-        telemData.start()
+        pixhawk.start()
```
Comment (avesk) [inline obc.py]: Nit-pick: You've got a couple extra lines of white space between these two functions. Normally, there should only be 1

Comment (logankuzyk) [reply obc.py]: That's a relief. I can go through and fix that right away.

File: apps/Routes/routes.py [hunk 8]
```diff
@@ -2,7 +2,7 @@
 from apps.Timelapse.routes import routes as tRoutes
 from apps.ImageService.routes import routes as isRoutes
 from apps.PiCam.routes import routes as pcRoutes
-from apps.TelemData.routes import routes as telemRoutes
+from apps.Pixhawk.routes import routes as telemRoutes
```
Comment (avesk) [inline apps/Routes/routes.py]: Non-blocking, but, @dragonprevost Do we even use this Routes file anymore since turfing `tornado`?

**Comment by dragonprevost** [comment]
Great work @logankuzyk, this issue was bigger than I had originally intended but I am sure you have gained a good sense of how obc works now!

File: apps/Routes/routes.py (hunk 8 above)
Comment (dragonprevost) [reply apps/Routes/routes.py]: Yeah this should get wiped once merged with master. I will run a simulated merge to make sure we don't get too many conflicts

Comment (dragonprevost) [reply apps/Routes/routes.py]: I just looked into this and the routes application still exists in master but lets just delete it after we merge this PR...

**Pull request merged**